components/__init__.py

```diff
@@ -342,7 +342,7 @@ def define_components(reqs):
     retriever = GitRepoRetriever("https://github.com/spdk/spdk.git", True)
     reqs.define('spdk',
                 retriever=retriever,
-                commands=['./configure --prefix="$SPDK_PREFIX" --with-shared ' \
+                commands=['./configure --prefix="$SPDK_PREFIX" ' \
```

Comment:
So, partially reverting daos-stack/scons_local#25?

I suppose this is what @jolivier23 was [trying to head off](https://github.com/daos-stack/scons_local/pull/25#pullrequestreview-260142575) (you need to expand the outdated comment in the link to see his comment).

Reply:
Yes.